.github/workflows/integrations_tests.yml

```diff
@@ -0,0 +1,37 @@
+name: integration tests
+
+on:
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: 'Branch to test on'
+        required: true
+      pytorch_nightly:
+        description: 'Whether to test integration tests (diffusers + transformers)'
+        required: false
+        default: false
+
+jobs:
+  run_transformers_integration_tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event.inputs.branch }}
+          repository: ${{ github.event.pull_request.head.repo.full_name }}
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.10"
+          cache: "pip"
+          cache-dependency-path: "setup.py"
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install .[test]
+          cd .. && git clone https://github.com/huggingface/transformers.git && cd transformers/
```

Comment:
I wonder if we need an option for transformers `main` vs latest release too?

Reply:
Makes sense yes, sounds great !